### DISCUSSION TITLE
Import/Export Language - Import/Export translated application content strings (Fields, etc.)

### DISCUSSION THREAD

**Discussion by DmitriyVdE** [pnp/pnpcore#1241] [Q&A]
Hi

SharePoint Online has an option to add/enable multiple languages which allows users to see the UI in their own language. This also makes it possible to import and export these translations within the Site Settings under Site Administration and Import/Export Translations. This is great, but as far as I can find this is now a manual process that has to be done within SharePoint itself through the front-end.

I found that it used to be possible through some kind of API call [here in the SP2010 docs](https://learn.microsoft.com/en-us/previous-versions/office/developer/sharepoint-2010/ff800886(v=office.14)#adding-translations-to-user-resources), but that doesn't seem to be the case anymore. I also can't find anything about this in the PnP.Core documentation.

Is this true or am I missing something? Thanks in advance!

Greetings
-Dmitriy

**Comment by jansenbe** [comment]
@DmitriyVdE : using those 'Resources' (e.g. `TitleResource`) does work in CSOM (and probably also in REST), but we did not yet work on implementing this in PnP Core SDK. I'm making an issue of this to track that need.
